From 59e1d963525ab9f624852ce8a0a5683b398c485b Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Mon, 14 Oct 2019 19:30:33 +0900
Subject: [PATCH 1/1] bin: add opt for waiting pri in startup script

This update is to add option `--wait-pri` for `bin/start.sh`.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 bin/start.sh | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/bin/start.sh b/bin/start.sh
index f39d5f6..2f4e31a 100755
--- a/bin/start.sh
+++ b/bin/start.sh
@@ -64,7 +64,6 @@ echo "Start spp_primary"
 start_spp_pri
 
 if [ ! ${DRY_RUN} ]; then
-    echo "Waiting for spp-ctl is ready ..."
-    sleep 1
-    python3 ${SPP_DIR}/src/spp.py -b ${SPP_HOST_IP}
+    sleep 1  # wait for spp-ctl is ready
+    python3 ${SPP_DIR}/src/spp.py -b ${SPP_HOST_IP} --wait-pri
 fi
-- 
2.17.1

